xcube/core/store/fs/impl/dataset.py

```diff
@@ -160,9 +162,12 @@ def get_open_data_params_schema(self, data_id: str = None) \
             ZARR_OPEN_DATA_PARAMS_SCHEMA
         )
 
-    def open_data(self, data_id: str, **open_params) -> xr.Dataset:
+    def open_data(self,
+                  data_id: str,
+                  root: Optional[str] = None,
```

Comment:
I think you should remove this here. It is later superseded by the root from load_fs, however, that one will always be None as root is not in open_params.

Reply:
Agreed!